exporter/signalfxexporter/config.go

```diff
@@ -104,6 +104,9 @@ type Config struct {
 	// NonAlphanumericDimensionChars is a list of allowable characters, in addition to alphanumeric ones,
 	// to be used in a dimension key.
 	NonAlphanumericDimensionChars string `mapstructure:"nonalphanumeric_dimension_chars"`
+
+	// MaxConnections is used to set a limit to the maximum idle HTTP connection the exporter can keep open.
+	MaxConnections uint `mapstructure:"max_connections"`
```

Comment:
Not sure I see the overall benefit of converting to an int, especially since negative values are significant in the transport.  Could be helpful to avoid edge cases by using an int field.

Reply:
I have added suggested changes. Please review the same.